src/globus_cli/commands/timer/create/__init__.py

```diff
@@ -4,7 +4,6 @@
 @group(
     "create",
     short_help="Submit a Timer job",
-    hidden=True,
```

Comment:
This is a good place to note or remind people: the code here existed and was even in existing CLI releases, but was marked `hidden=True`, meaning it wasn't published in docs or visible in helptext. It was latent, but not officially released or supported.

So this change marks our last chance for highly backwards incompatible changes to the command before we consider it subject to our support guarantees.